ckanext/reclineview/theme/public/recline_view.js

```diff
@@ -63,17 +63,22 @@ this.ckan.module('recline_view', function (jQuery, _) {
       var query = new recline.Model.Query();
       query.set({ size: reclineView.limit || 100 });
       query.set({ from: reclineView.offset || 0 });
-
+      
+      var windowThere = false;
       try {
         if (window.parent.ckan.views && window.parent.ckan.views.filters) {
-          var defaultFilters = reclineView.filters || {},
-              urlFilters = window.parent.ckan.views.filters.get(),
-              filters = $.extend({}, defaultFilters, urlFilters);
-          $.each(filters, function (field,values) {
-            query.addFilter({type: 'term', field: field, term: values});
-          });
+          windowThere = true;
         }
       } catch(e) {}
+      var urlFilters = {};
+      if (windowThere) {
```

Comment:
You can move the assignment to `urlFilters` inside the `try`-clause, that way you don't need to introduce `windowThere`.